<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fornecedor</class>
 <widget class="QDialog" name="Fornecedor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbApagar">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apagar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFornecedor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSalvarEditar">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbinstituicaoBancaria">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>InstituiçãoBancária</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteDescricao">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>511</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbwFornecedor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>511</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
